nsxt/resource_nsxt_logical_dhcp_port.go

```diff
@@ -0,0 +1,182 @@
+/* Copyright © 2017 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	api "github.com/vmware/go-vmware-nsxt"
+	"github.com/vmware/go-vmware-nsxt/manager"
+	"log"
+	"net/http"
+)
+
+func resourceNsxtLogicalDhcpPort() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtLogicalDhcpPortCreate,
+		Read:   resourceNsxtLogicalDhcpPortRead,
+		Update: resourceNsxtLogicalDhcpPortUpdate,
+		Delete: resourceNsxtLogicalDhcpPortDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
Perhaps we should verify this port has dhcp attachment on import (read below only verifies attachment exists)

Reply:
Thanks. I'll add this check to the Read function